3515/2/commands/checkin.rb

```diff
@@ -0,0 +1,85 @@
+require_relative '../helper_modules/directory_maker'
+require_relative '../helper_modules/file_write'
+module CheckinCommand
+  def checkin!(*)
+    if user_authorized?
+      checkin_false?
+    else
+      respond_with :message, text: "Извините, но ваш аккаунт не авторизован\nДля авторизации введите
+      команду /start"
+    end
+  end
+
+  def user_authorized?
+    !(User[from['id']]).to_s.empty?
```

Comment:
Not really readable one-liner IMO. Maybe there's a way to make it more clear?

Reply:
I'd wrap `User[from['id']]` into a private method (for example `current_user`) so all the usages of that constructions would be a bit more clear. BTW think of a way to avoid parenthesis here